src/HybridDb/Queue/DequeueCommand.cs

```diff
@@ -77,10 +77,14 @@ and cast('/' + Version + '/' as hierarchyid) <= cast('/' + @Version + '/' as hie
 
             var metadata = (Dictionary<string, string>)deserializer(msg.Metadata, typeof(Dictionary<string, string>));
 
-            return new HybridDbMessage(msg.Id, deserializer(msg.Payload, type), msg.Topic, msg.Order)
+            var result= new HybridDbMessage(msg.Id, deserializer(msg.Payload, type), msg.Topic, msg.Order)

```

Comment:
Man kunne vel godt stadig bruge ctor til CorrelationId... det er jo egentlig Enqueue, der bestemmer, at  CorrelationId = Id, og hvis man først er nede at bruge HybridDbMessage direkte, så er det nok fordi man gerne vil bestemme det hele selv?

Reply:
Vi kan jo stadig godt beholde OVerrideCorrelationId hvis den er nødvendig?

Reply:
Det er p.t. ikke et behov for at sætte CorrelationID udefra - derfor har jeg udeladt den.